sass/includes/_event-details-banner.scss

```diff
@@ -0,0 +1,14 @@
+@use "@nationalarchives/frontend/nationalarchives/tools/colour";
+@use "@nationalarchives/frontend/nationalarchives/tools/media";
+
+.event-details-banner {
+  &__cta {
+    margin: 0;
+    display: block;
+
+    @include media.on-larger-than-mobile {
+      background-color: colour.brand-colour("pastel-pink");
```

Comment:
I think this could use the same `@include colour.light-accent-background;` that AJ suggested for my merge request, with `{% block html_class %}tna-template--pink-accent{% endblock %} ` in the template.